### DISCUSSION TITLE
AI聊天機器人須學會求助

### DISCUSSION THREAD

**Discussion by Raiden1121** [1101-GS4719-C/Python-Community-GS4719-C#2091] [General]
https://www.cna.com.tw/news/ait/202409010127.aspx?topic=3437
https://futurecity.cw.com.tw/article/2990
AI一直讓人有一種無所不知的感覺，然而生程式AI卻時常給出錯誤答案。
我很認同文章所說的「生成式人工智慧讓企業節省大量時間與金錢，但模型必須學會承認自身並非無所不能。」
AI的出錯，常常讓使用者誤以為信，導致錯誤的訊息以訛傳訛。AI出錯的原因，常常是因為語言模型的訓練資料可能存在一定的模型偏見，因此我認為AI的學習資料因該要人為把關，避免將錯誤或帶有偏見的資訊傳遞出去。

**Comment by Piyenparty** [comment]
所以我們真的需要培養一些媒體識讀的能力，也不能直接摳貼AI給我的內容，而是應該多用心去思考一下內容是否正確，不然資料的流通速度是很可怕的，以訛傳訛的話真的有一天會造成錯誤資訊的潰堤
